lib/http.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * The HTTP methods
+ */
+export enum HttpMethod {
+  GET = "GET",
+  POST = "POST",
+  PUT = "PUT",
+  PATCH = "PATCH",
+  DELETE = "DELETE",
+  OPTIONS = "OPTIONS",
+}
```

Comment:
Do we want to have this in two locations? I added the same contents at `utils/http.ts`, so wondering if I should delete the one I have and use the one from this location (assuming this will be merged before my branch).

Reply:
Let's use this one?

Reply:
Fine by me.